src/Protocol/Diaspora.php

```diff
@@ -3203,59 +3173,35 @@ public static function sendUnshare(array $owner, array $contact): int
 	/**
 	 * Checks a message body if it is a reshare
 	 *
-	 * @param string $body     The message body that is to be check
-	 * @param bool   $complete Should it be a complete check or a simple check?
+	 * @param array $item The message body that is to be check
 	 *
-	 * @return array|bool Reshare details or "false" if no reshare
+	 * @return array Reshare details or "false" if no reshare
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 * @throws \ImagickException
 	 */
-	public static function isReshare(string $body, bool $complete = true)
+	public static function isReshare(array $item): array
```

Comment:
This method should be renamed now that it always return an array.
```suggestion
	public static function getShareRootDetails(array $item): array
```

Reply:
Renamed